static/_redirects

```diff
@@ -209,5 +209,7 @@ https://events.prideinlondon.org/*     /events      301!
 /event/queer-lady-m-200420-2XaZ1svjqz6TQ9YX8ytmPa /event/queer-lady-m-2XaZ1svjqz6TQ9YX8ytmPa 301!
 /event/proud-coming-out-is-everyones-journey-260620-6FlCfDAqYxAtoC0lEGtkju /event/proud-coming-out-is-everyones-journey-6FlCfDAqYxAtoC0lEGtkju 301!
 /event/proud-coming-out-is-everyones-journey-270620-6FlCfDAqYxAtoC0lEGtkju /event/proud-coming-out-is-everyones-journey-6FlCfDAqYxAtoC0lEGtkju 301!
+/events/prides-got-talent /pgt 301!
+/events/prides-got-talent/audition /pgt/audition 301!
```

Comment:
Redirect for `/pgt` already exists - we might end up in a continuous loop here 😱

Reply:
Continuous loops ❤️